packages/liveblocks-core/src/__tests__/immutable.test.ts

```diff
@@ -82,25 +82,25 @@ export async function prepareStorageImmutableTest<
     { isDeep: true }
   );
 
-  function assert(
+  function expectStorageAndStateInBothClients(
     data: ToJson<TStorage>,
     itemsCount?: number,
     storageOpsCount?: number
   ) {
-    assertStorage(data);
+    expectStorageInBothClients(data);
 
-    if (itemsCount) {
+    if (itemsCount !== undefined) {
```

Comment:
This is nitpicky but in general I prefer to use typeof var !== "undefined" as it works more consistently, although in this case it makes no difference.

Reply:
You mean to protect against cases where someone would define a global variable called `undefined`?

Reply:
Haha, yes. I don’t know how we did things without TypeScript 😬